config/config.example.toml

```diff
@@ -28,6 +28,7 @@
 # #proxy = "" # env variable: PROXY
 # # Enable YouTube new video format UMP
 # ump = false
+# video_fetch_chunk_size_mb = 5 # size of chunks to request from google servers for rate limiting reductions
```

Comment:
would probably be a good idea to start to create a dedicated section for videoplayback proxy like:
```
[networking.videoplayback]
ump = false
video_fetch_chunk_size_mb = 5
```

I have a feeling we are going to add more parameters. So the earlier we do it, the better it is.

Could you do that? You can also move "ump", nobody is probably using it yet.

Thanks.

Reply:
Done, and rebased against the latest